### ISSUE TITLE
agda-categories should list compatibility versions on README

### ISSUE THREAD

**Issue by Boarders** [agda/agda-categories#414]
Similar to the cubical library it would be nice if `agda-categories` listed the compatibilities of the versions of `agda` it works with. See the `compiling and installing` section of this README: https://github.com/agda/cubical

**Comment by JacquesCarette** [comment]
Agreed. I'd welcome a PR that documented that. [I just know I might not get the time to do this for a couple of weeks.]
